charts/fulcio/Chart.yaml

```diff
@@ -5,8 +5,8 @@ description: |
 
 type: application
 
-version: 2.3.9
-appVersion: 1.4.0
+version: 2.3.10
+appVersion: 1.4.1
```

Comment:
pls bump the annotations below, as well as createcerts to v0.6.8

Reply:
done